ipld/kamt/src/kamt.rs

```diff
@@ -81,6 +82,17 @@ where
         }
     }
 
+    /// Construct kamt with a bit width
+    pub fn new_with_bit_width(store: BS, bit_width: u32) -> Self {
```

Comment:
Let's drop this for now. We have it on the HAMT for historical reasons, but `new_with_config` is more general.